src/main.js

```diff
@@ -47,17 +78,29 @@ module.exports = {
     )
 
     this.subscriptions.add(atom.workspace.observeTextEditors((editor) => {
-      editor.onDidSave(() => {
+      editor.onDidSave(async () => {
         const validScope = editor.getCursors().some(cursor =>
           cursor.getScopeDescriptor().getScopesArray().some(scope =>
             scopes.includes(scope)))
         if (validScope && atom.config.get('linter-eslint.fixOnSave')) {
+          if (this.worker === null) {
+            await waitOnIdle()
+          }
+          if (!path) {
+            path = require('path')
+          }
+          if (!isConfigAtHomeRoot) {
+            isConfigAtHomeRoot = require('./is-config-at-home-root')
```

Comment:
Fix-on-save is currently broken, with an error message that `isConfigAtHomeRoot` isn't a function.  It seems that you need to use `isConfigAtHomeRoot = require('./is-config-at-home-root').default`, although to be honest, I'm not entirely clear on the "right" way to interop `require` statements with `export`.  Too bad we don't have `import()` yet.